Code/ForceField/MMFF/AngleBend.cpp

```diff
@@ -1,4 +1,5 @@
 // $Id$
```

Comment:
While you're here, please remove these ancient SVN artifacts (both here and in the other files)

Reply:
Done